.clang-tidy

```diff
@@ -1,30 +1,39 @@
 ---
 # turn off readability-braces-around-statements to allow single line statement like 'if (x == y) doSomething();'
-Checks:          '-*,cppcoreguidelines-*,google-*,readability-*,modernize-*,-readability-braces-around-statements,-google-runtime-references,-cppcoreguidelines-pro-type-reinterpret-cast'
-WarningsAsErrors: ''
+Checks: >
+  -*,
+  cppcoreguidelines-*,
+  google-*,
+  readability-*,
+  modernize-*,
+  bugprone-*,
+  -readability-braces-around-statements,
+  -google-runtime-references,
+  -cppcoreguidelines-pro-type-reinterpret-cast,
+  -modernize-use-trailing-return-type
```

Comment:
I think this shall be documented more carefully. Although multiline string vastly imporve readability, it makes easier to make mistake. 
1. We shall indicate it is a multiline string here
2. We shall explicitly document that comment must not move into the string
3. We shall explicitly document that why the specific checks are disable
4. the last line should be terminated with a comma, that is `-modernize-use-trailing-return-type,`